ffi/hello_world/hello.dart

```diff
@@ -13,8 +13,8 @@ typedef HelloWorld = void Function();
 main() {
   // Open the dynamic library
   var path = "./hello_library/libhello.so";
-  if (Platform.isMacOS) path = "./hello_library/libhello.dylib";
-  if (Platform.isWindows) path = "hello_library\libhello.dll";
+  if (Platform.isMacOS) path = './hello_library/libhello.dylib';
+  if (Platform.isWindows) path = r'hello_library\Debug\hello.dll';
```

Comment:
do the other 2 samples need this change too?

Reply:
Yup, I'll do the two other samples in follow up PRs.